public/kibi_timeline.js

```diff
@@ -80,6 +81,19 @@ define(function (require) {
         if (!timeline) {
           // create a new one
           timeline = new vis.Timeline($element[0]);
+          var timezone = config.get('dateFormat:tz');
+          if (timezone !== 'Browser') {
+            var utcOffset = null;
+            if (timezone.substring(0,7) === 'Etc/GMT') {
+              utcOffset = timezone.substring(7, 8) +
+              (timezone.substring(8) < 10 ? '0' : '') + timezone.substring(8) + ':00';
```

Comment:
Here you are comparing strings with integers `timezone.substring(8) < 10`. What do you want to do ?